data/net/minecraft/client/gui/narration/ScreenNarrationCollector.mapping

```diff
@@ -1,4 +1,12 @@
 CLASS net/minecraft/client/gui/narration/ScreenNarrationCollector
+	METHOD lambda$collectNarrationText$2 (ZLjava/util/function/Consumer;Lnet/minecraft/client/gui/narration/ScreenNarrationCollector$EntryKey;Lnet/minecraft/client/gui/narration/ScreenNarrationCollector$NarrationEntry;)V
+		ARG 3 key
+		ARG 4 entry
+	METHOD update (Ljava/util/function/Consumer;)V
+		ARG 1 updater
+	CLASS 1
```

Comment:
Class 1? 
@sciwhiz12 synthethic bs?

Reply:
Nope, anonymous classes start with numbers (a feature of Java, since classes in the language source code must begin with letters).